doc/news/changes/minor/20250201Shi

```diff
@@ -0,0 +1,5 @@
+Fix: Replace GeometryInfo<dim>::face_indices() with
+cell->reference_cell().face_indices() in GridGenerator::extract_boundary_mesh()
+to correctly handle simplex meshes and prevent invalid face index access.
```

Comment:
@bangerth I don't disagree with merging this but this changelog isn't right: there's still more work to do here. We don't yet correctly handle simplex meshes.

Reply:
True. @QY-Shi Would you be willing to remove the changelog entry, or reword it?